dockerExtension.ts

```diff
@@ -80,6 +88,17 @@ export function activate(ctx: vscode.ExtensionContext): void {
     ctx.subscriptions.push(vscode.commands.registerCommand('vscode-docker.compose.down', composeDown));
     ctx.subscriptions.push(vscode.commands.registerCommand('vscode-docker.system.prune', systemPrune));
 
+    ctx.subscriptions.push(vscode.commands.registerCommand('vscode-docker.createWebApp', async (context?: AzureImageNode | DockerHubImageNode) => {
+        const wizard = new WebAppCreator(outputChannel, azureAccount, context);
+        const result = await wizard.run();
+
+        if (result.status === 'Completed') {

```

Comment:
Can we just delete this statement entirely?

Reply:
removed...